### PR TITLE
View inidividual article

### PR DESCRIPTION
**Pull request by CarageaAlexandru**
Added functionality for viewing Single Card

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -14,8 +15,12 @@ function App() {
...
 					<Route path="/" element={<Home />}></Route>
 					<Route path="/articles" element={<Articles />}></Route>
-          <Route
-						path="/api/articles/:topic"
+					<Route
+						path="/api/articles/:article_id"
+						element={<SingleArticleCard />}
+					></Route>
+					<Route
+						path="/api/articles/topic/:topic"
```
Comment (jimstev2447) [inline src/App.js:23]: should these paths be /api? 
I might suggest:
/articles/:article_id
/topics/:topic

File: src/components/SingleArticleCard.jsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+const SingleArticleCard = () => {
+	const [singleArticle, setSingleArticle] = useState({});
+	const [loading, setLoading] = useState(true);
+	const { article_id } = useParams();
+	const { id, author, body, comment_count, created_at, title, topic, votes } =
+		singleArticle;
+
+	useEffect(() => {
+		const url = `https://caragea-nc-news-backend.herokuapp.com/api/articles/${article_id}`;
+		fetch(url)
```
Comment (jimstev2447) [inline src/components/SingleArticleCard.jsx:19]: dont forget to think about your errors

File: src/components/SingleArticleCard.jsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+	const hour = getHour(created_at);
+	return (
+		<div className="container">
+			<Stack gap={3}>
+				<div className="bg-light border">
+					<Card
+						style={{ width: "18rem" }}
+						className="text-dark d-inline"
+						key={id}
+					>
```
Comment (jimstev2447) [inline src/components/SingleArticleCard.jsx:56]: image
you have used react bootstrap here but I think the way this is set out might be a little confusing if the article was very long, all the information would be below the body of the text, IDK if its intentional but it seems a little strange, also what are Second Item and Third Item????

Its also quite div soupy

File: src/components/SingleArticleCard.jsx (hunk 2 above)
Comment (CarageaAlexandru) [reply src/components/SingleArticleCard.jsx:19]: Will start implementing error handling after adding functionality for all the tasks.

File: src/App.js (hunk 1 above)
Comment (CarageaAlexandru) [reply src/App.js:23]: Sure, makes more sense to have them without /api
Will change them.

File: src/components/SingleArticleCard.jsx (hunk 3 above)
Comment (CarageaAlexandru) [reply src/components/SingleArticleCard.jsx:56]: I understand, will try to add a fixed-size component.
The second item I left it there for posting a comment form and the third will be the container for rendering the comments associated with the specific article, hope will get that working.
Will try to use more semantic tags.

Comment (CarageaAlexandru) [reply src/components/SingleArticleCard.jsx:56]: Thanks for the feedback

**Pull request merged**